drivers/modem/hl7800.c

```diff
@@ -2466,6 +2482,75 @@ static bool on_cmd_operator_index_query(struct net_buf **buf, uint16_t len)
 	return true;
 }
 
+/* There can be multiple responses from a single command.
+ * %MEAS: EARFCN=5826, CellID=420, RSRP=-99, RSRQ=-15
+ * %MEAS: EARFCN=6400, CellID=201, RSRP=-93, RSRQ=-21
+ */
+static bool on_cmd_survey_status(struct net_buf **buf, uint16_t len)
+{
+	struct net_buf *frag = NULL;
+	char response[] =
+		"EARFCN=XXXXXXXXXXX, CellID=XXXXXXXXXXX, RSRP=-XXX, RSRQ=-XXX";
```

Comment:
IMHO it would be better to have
```
	char response[sizeof("EARFCN=XXXXXXXXXXX, CellID=XXXXXXXXXXX, RSRP=-XXX, RSRQ=-XXX")];
```
This way we do not need to store the string anywhere as the `sizeof()` is calculated at compile time. This is the pattern we have been doing it in other part of the code base.